lib/pages/login-page.js

```diff
@@ -43,6 +43,40 @@ export default class LoginPage extends BaseContainer {
 		} );
 	}
 
+	use2FAMethod( twoFAMethod ) {
+		let actionSelector;
+
+		if ( twoFAMethod === 'sms' ) {
+			actionSelector = By.linkText( 'Code via text message' );
+		} else if ( twoFAMethod === 'otp' ) {
+			actionSelector = By.linkText( 'Your authenticator app' );
+		} else if ( twoFAMethod === 'backup' ) {
+			actionSelector = By.linkText( 'I can\'t access my phone' );
+		}
+
+		if ( actionSelector ) {
+			return driverHelper.isElementPresent( this.driver, actionSelector ).then( ( actionAvailable ) => {
+				if ( actionAvailable ) {
+					return driverHelper.clickWhenClickable( this.driver, actionSelector );
+				} else {
+					return Promise.resolve();
```

Comment:
can just `return;` here

Reply:
Now that I think about it, shouldn't we actually reject here?

Reply:
Yes we should 👍 thanks for noticing!

Reply:
Actually no, we should resolve here.
If the action is not available we're assuming that we're already on the right page.
The case where the option is not available is not tested right now!